design/www/app/chatt.constant.js

```diff
@@ -2,7 +2,7 @@
     'use strict';
     angular.module('starter')
         .constant('Configurations', {
-            api_url: 'http://192.168.1.149:3000/api',
+            api_url: 'http://144.76.34.244:3033/api/users',
```

Comment:
why are u putting /users here? if something api has /friends or /chat how will you do then?